imx95_evk a55: missing sram0 DT label breaks arm64_high_addresses tests

### Describe the bug

The Twister (2) build has been blocked due to a build failure “devicetree error: ./high_address.overlay:1 (column 1): parse error: undefined node label 'sram0'”


### Regression

- [ ] This is a regression.

### Steps to reproduce

command ：
west build -p -b imx95_evk/mimx9596/a55 tests/arch/arm64/arm64_high_addresses -T arch.arm64.high_addr.high_sram_equal_vm.picolibc

Error Log：

Call Stack (most recent call first):
  /__w/zephyr/zephyr/cmake/modules/dts.cmake:438 (dts_edt_pickle)
  /__w/zephyr/zephyr/cmake/modules/zephyr_default.cmake:133 (dts_init)
  /__w/zephyr/zephyr/cmake/modules/zephyr_default.cmake:133 (cmake_language)
  /__w/zephyr/zephyr/share/zephyr-package/cmake/ZephyrConfig.cmake:66 (include)
  /__w/zephyr/zephyr/share/zephyr-package/cmake/ZephyrConfig.cmake:92 (include_boilerplate)
  CMakeLists.txt:5 (find_package)
-- Configuring incomplete, errors occurred!

INFO    - /__w/zephyr/zephyr/twister-out/imx95_evk_mimx9596_a55_smp/zephyr_gnu/tests/arch/arm64/arm64_high_addresses/arch.arm64.high_addr.high_sram_low_vm/build.log
**ERROR   - CMake build failure: /__w/zephyr/zephyr/tests/arch/arm64/arm64_high_addresses for imx95_evk/mimx9596/a55/smp
INFO    - 861/968 imx95_evk/mimx9596/a55/smp arch.arm64.high_addr.high_sram_high_vm.picolibc   ERROR CMake build failure (build** <zephyr/gnu>)
INFO    - /__w/zephyr/zephyr/twister-out/imx95_evk_mimx9596_a55_smp/zephyr_gnu/tests/arch/arm64/arm64_high_addresses/arch.arm64.high_addr.high_sram_high_vm.picolibc/build.log
ERROR   - Loading Zephyr default modules (Zephyr base).
-- Application: /__w/zephyr/zephyr/tests/arch/arm64/arm64_high_addresses
-- CMake version: 3.28.3
-- Found Python3: /opt/python/venv/bin/python3 (found suitable version "3.12.3", minimum required is "3.12") found components: Interpreter 
-- Cache files will be written to: /root/.cache/zephyr
-- Zephyr version: 4.4.99 (/__w/zephyr/zephyr)
-- Found west (found suitable version "1.5.0", minimum required is "0.14.0")
-- Board: imx95_evk, qualifiers: mimx9596/a55/smp
-- Found host-tools: zephyr 1.0.1 (/opt/toolchains/zephyr-sdk-1.0.1)
-- Found toolchain: zephyr 1.0.1 (/opt/toolchains/zephyr-sdk-1.0.1)
-- Found Dtc: /opt/toolchains/zephyr-sdk-1.0.1/hosttools/sysroots/x86_64-pokysdk-linux/usr/bin/dtc (found suitable version "1.7.0", minimum required is "1.4.6") 
-- Found BOARD.dts: /__w/zephyr/zephyr/boards/nxp/imx95_evk/imx95_evk_mimx9596_a55_smp.dts
-- Found devicetree overlay: high_address.overlay
devicetree error: ./high_address.overlay:1 (column 1): parse error: undefined node label 'sram0'
CMake Error at /__w/zephyr/zephyr/cmake/modules/dts.cmake:324 (execute_process):
  execute_process failed command indexes:

Full log link : https://github.com/zephyrproject-rtos/zephyr/actions/runs/25716728672/job/75508931695?pr=108710

### Relevant log output

```shell
Twister(2) build pass
```

### Impact

Functional Limitation – Some features not working as expected, but system usable.

### Environment

- Board: IMX95EVK
- Test case: tests/arch/arm64/arm64_high_addresse
- SD card: 64GB

### Additional Context

_No response_

--- a/tests/arch/arm64/arm64_high_addresses/testcase.yaml
+++ b/tests/arch/arm64/arm64_high_addresses/testcase.yaml
@@ -1,7 +1,6 @@
 common:
   arch_allow: arm64
-  platform_exclude:
-    - rza3ul_smarc
+  platform_allow: qemu_cortex_a53
   tags:
     - arm
     - userspace

